src/strategies/radicle-funding-nft/index.ts

```diff
@@ -0,0 +1,53 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+const FUNDING_SUBGRAPH_URL = {
+  '1': 'https://api.studio.thegraph.com/query/9578/funding-subgraph-v4/v0.0.2'
+};
+
+export async function strategy(
```

Comment:
index.ts should also export author and version
```
export const author = 'AmirSarraf';
export const version = '0.1.0';
```

example:
https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/erc20-balance-of/index.ts

Reply:
done